backend/database/seeders/DatabaseSeeder.php

```diff
@@ -19,8 +21,15 @@ public function run()
             'username' => 'regularUser',
             'email' => 'regularuser@ccrproject.dev',
             'name' => 'Regular User',
-            'password' => Hash::make('regularPassword!@#')
+            'password' => Hash::make('regularPassword!@#'),
         ]);
-        
+
+        $user = User::factory()->create([
+            'username' => 'systemAdminUser',
+            'email' => 'adminuser@ccrproject.dev',
+            'name' => 'System Admin User',
```

Comment:
Should we change the term from 'Application Admin' to 'System Admin'? 

I notice other team members keep defaulting to "system" when describing this role as well.

Reply:
Whether it's application or system doesn't really matter to me.  I can go either way.  I will say that I think the role titles should be APPLICATION_ADMIN rather than "Application Admin" (caps snakecase over human strings)

Reply:
I updated the test user to be title application administrator.